ui/app/src/index.html

```diff
@@ -2,6 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
+    <link rel="icon" href="/favicon.ico" type="image/png/ico" />
```

Comment:
@Nexucis so the favicon is showing for me [here](http://localhost:3000/favicon.ico) and I made the required webpack changes. Not sure if this is just a local dev issue or what, but I need to finish up some other tasks, so I can come back to this later or if you want to take a quick look to see if it works for you

Reply:
it works for me when the binary is built and when I'm using the webpack dev environment :).
So all good on my side ! Thank you for doing it @sjcobb !!

Maybe you just need to hard refresh the page

Reply:
Okay great! Alright this should be ready for review now, I had to add a placeholder link in the README h1 (or else it would try to link to the SVG file for some reason), but we can always change that href later when we have a docs site